OpenEXR/IlmImf/ImfFastHuf.cpp

```diff
@@ -395,7 +395,7 @@ FastHufDecoder::buildTables (Int64 *base, Int64 *offset)
                 _tableCodeLen[i] = codeLen;
 
                 Int64 id = _ljOffset[codeLen] + (value >> (64 - codeLen));
-                if (id < _numSymbols)
+                if (id < Int64(_numSymbols))
```

Comment:
static_cast<Int64>

Reply:
I've converted all constructor casts to static_cast